src/status.rs

```diff
@@ -274,10 +277,10 @@ pub async fn get_status(sentry: &SentryApi, channel: &Channel) -> Result<Status,
     let disconnected = is_disconnected(&channel, &messages);
 
     // impl: isInvalid
-    let rejected_state = is_rejected_state();
+    let rejected_state = is_rejected_state(&messages, &latest_new_state[0]);
```

Comment:
usually this can fail (panic! and crash the program, this is why we have `Option` - `vec.get(0)`) :grimacing: since you are calling `[0]`, but there might not be a new state in the response